SDPi_Supplement/asciidoc/sdpi-supplement.adoc

```diff
@@ -1,6 +1,6 @@
 = Service-oriented Device Point-of-care Interoperability (SDPi) Technical Framework
 :doctype: book
-:preface-title: Preface
+:preface-title: Foreword

```

Comment:
This can be removed as the HTML output does not use the preface-title variable anywhere.

Reply:
OK ... I put it in there because you put it in there :-)  Outtahere!